addons/api/mirage/config.js

```diff
@@ -431,9 +431,16 @@ function routes() {
       return hostSets.where({ hostCatalogId });
     }
   );
+
   this.post('/host-sets', function ({ hostCatalogs, hostSets }) {
     const attrs = this.normalizedRequestAttrs();
     const hostCatalog = hostCatalogs.find(attrs.hostCatalogId);
+    if (hostCatalog.plugin) {
```

Comment:
currently there's an issue when you update a newly created plugin type host set due to not attaching `plugin` info when we create a plugin host set.

Reply:
None blocking comment: Can we add a comment to explain why we are doing this? It will help maintenance. I.e: Adding attributes to avoid error at updating time

Reply:
sure I can add a comment, but basically the creation of a new dynamic plugin in mirage doesn't work as expected. In mirage, we aren't attaching the plugin information similar to actual APIs